library/userHelper.ts

```diff
@@ -40,6 +40,20 @@ export async function getUserBalance(userId: string): Promise<string> {
     });
 }
 
+export async function getUserName(userId: string): Promise<string> {
```

Comment:
looks like the `.catch` will result in nothing being returned
```suggestion
export async function getUserName(userId: string): Promise<string | undefined> {
```